core/web-assets/src/main/assets/js/apps/onms-classifications/index.js

```diff
@@ -312,6 +314,59 @@ const confirmTopoverTemplate = require('./views/modals/popover.html');
                 });
             };
 
+            // for drag and drop of rules (redefining position)
+            $scope.sortableRules = {
+
+                start: function(e, ui) {
+                    // remember old index before moving
+                    $(ui.item).data('oldIndex', ui.item.index());
+                },
+                stop: function(e, ui) {
+
+                    // 1.) Check Precondition: not read only group
+                    if($scope.group.readOnly) {
+                        ui.item.parent().sortable('cancel');
+                        $window.alert('this group cannot be sorted, it is read only');
```

Comment:
Would it be worth preventing the rules from being reordered in the first place instead?

Reply:
I tried but I couldn't find a good way to do so with my limited Javascript knowledge (but I am sure there is). I can take another look...

Reply:
We should not allow for changing the position in read only groups.

Reply:
fixed